unit3MidUnitAssessment/unit3MidUnitAssessment/AlbumTableViewCell.swift

```diff
@@ -0,0 +1,52 @@
+//
+//  AlbumTableViewCell.swift
+//  unit3MidUnitAssessment
+//
+//  Created by Erica Y Stevens on 10/29/16.
+//  Copyright © 2016 Erica Stevens. All rights reserved.
+//
+
+import UIKit
+
+class AlbumTableViewCell: UITableViewCell {
+
+    @IBOutlet weak var albumCoverImageView: UIImageView!
+    @IBOutlet weak var albumTitleLabel: UILabel!
+ 
+    override func awakeFromNib() {
+        super.awakeFromNib()
+    
+        // Initialization code
+    }
+
+    override func setSelected(_ selected: Bool, animated: Bool) {
+        super.setSelected(selected, animated: animated)
+
+        // Configure the view for the selected state
+    }
+    
+
+}
+
+//Source: https://www.youtube.com/watch?v=XFvs6eraBXM
+extension UIImageView {
```

Comment:
Why is your extension of UIImageView inside custom tableview cell? Have it in its own extension file `UIImageView + loadImages` or something like that...